portfolio/src/main/webapp/script.js

```diff
@@ -126,24 +126,21 @@ function getComments(maxComments) {
 
     const commentsHistory = document.getElementById('comments-history');
     commentsHistory.innerHTML = '';
+    
+    comments.map(comments => constructCommentNode(comments)).forEach(node => commentsHistory.appendChild(node));
```

Comment:
You can be even more concise here by giving `map` just the name of the function, instead of a lambda. The default behavior is to call the provided function with each element in the first argument position, so if your function matches that signature (like `constructCommentNode` does), you can just give it the name of that function.

So: `comments.map(constructCommentNode)` is the same as the bit that you have now.

Reply:
Ah, this is the correct syntax for this - I forget this is possible in JS, I had been trying to do `comments.map(constructCommentNode(comments)` but ran into errors. Thanks! I'll update this.